src/rinq/attr_test.go

```diff
@@ -27,6 +27,11 @@ var _ = Describe("Attr", func() {
 			attr := rinq.Attr{Key: "foo", Value: "", IsFrozen: true}
 			Expect(attr.String()).To(Equal("!foo"))
 		})
+
+		It("escapes attributes that contain certain characters", func() {
+			attr := rinq.Attr{Key: "foo key", Value: "bar value"}
+			Expect(attr.String()).To(Equal("\"foo key\"=\"bar value\""))
```

Comment:
This is just a suggestion, but you could use a "raw string literal" here, instead of a double quoted string literal.

https://golang.org/ref/spec#String_literals